docs/control-structures.rst

```diff
@@ -22,6 +22,42 @@ there is in C and JavaScript, so ``if (1) { ... }`` is *not* valid Solidity.
 
 .. _function-calls:
```

Comment:
Please move that below. The section you added seems to be about function definitions.